docs/sdk-and-tools/indices/blocks.md

```diff
@@ -0,0 +1,93 @@
+---
+id: blocks
+title: blocks
+---
+
+The `_id` field for this index is composed of block hash hex encoded
+
+| Field                 | Description                                                                                                |
+|-----------------------|------------------------------------------------------------------------------------------------------------|
+| nonce                 | The nonce field represents sequence number of block.                                                       |
+| round                 | The round field represents the round when block was proposed and executed.                                 |
+| epoch                 | The epoch field represent the epoch when the block was proposed and executed.                              |
+| miniBlocksHashes      | The miniBlocksHashes field represents the hashed hex encoded of the miniblocks that was included in block. |
+| miniBlocksDetails     | The miniBlocksDetails field represents the details of all the miniblocks that was included in block.       |
+| notarizedBlocksHashes | The notarizedBlocksHashes field represents the hashes of the blocks the was notarized in current block.    |
+| proposer              | The proposer field represents the index of the validator that proposed this block.                         |
+| validators            | The validators field represents the indices on the validators that have signed for this block.             |
```

Comment:
The validators field is an array that contains the indices of the validators that signed the block. The indices are to be found in the `validators` index.

Reply:
done.